x-pack/plugins/serverless_search/public/application/components/connectors/connector_config/connector_index_name.tsx

```diff
@@ -80,6 +90,61 @@ export const ConnectorIndexName: React.FC<ConnectorIndexNameProps> = ({ connecto
         onChange={(name) => setNewIndexname(name)}
       />
       <EuiSpacer />
+      <EuiPanel hasBorder>
+        <EuiFlexGroup direction="column" justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiTitle size="s">
+              <h3>
+                {i18n.translate('xpack.serverlessSearch.connectors.config.preprocessData.title', {
+                  defaultMessage: 'Preprocess your data',
+                })}
+              </h3>
+            </EuiTitle>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiText>
+              <p>
+                <FormattedMessage
+                  id="xpack.serverlessSearch.connectors.config.preprocessData.description"
+                  defaultMessage="You can use ingest pipelines to preprocess data before indexing into Elasticseearch. Connectors clients use {clientIngestionPipeline} pipeline before indexing data into Elasticsearch"
+                  values={{
+                    clientIngestionPipeline: (
+                      <strong>
+                        {i18n.translate(
+                          'xpack.serverlessSearch.connectors.config.preprocessData.clientIngestionPipeline',
+                          {
+                            defaultMessage: 'search-default-ingestion',
```

Comment:
since this is the name of the default ingest pipeline I think we should 
1. use a constant for it
2. we should not translate it.

Reply:
maybe use code format too? 

`search-default-ingestion`

Reply:
Updated in 59e464a3c72c43b74155edf3e836369297bf9adc